support-workers/deploy-to-code.sh

```diff
@@ -0,0 +1,13 @@
+S3_BUCKET="membership-dist"
+S3_KEY="support/CODE/support-workers-typescript/support-workers.zip"
+LAMBDA_FUNCTIONS=(
+  "-CreatePaymentMethodLambda-"
+)
+echo "Uploading target/typescript/support-workers.zip to S3 (to update the zipfile before upload use the yarn deploy-to-code command)"
```

Comment:
if we want to make this script require less knowledge it could either call the yarn command for you, or if that's slow, it could check if the file exists and is up to date first.